### DISCUSSION TITLE
maps deactivated, map layer deleted

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#448] [Q&A] [answered]
I had it several times that the offline maps had been deactivated and map layer deleted and the design switched to Motorider when cruiser  started. I prefer the biker design and don't want to have this changed. It should keep what I have set. Don't know if it is connected with the recently downloaded maps.

Also after changing a design all map layers are deleted, why can't they be kept or at least restored when changing the design?

**Comment by devemux86** [comment, marked as answer]
@DolomitesTom thanks for the report.

What map was displayed, an online map?
This could happen when starting the app and the offline maps cannot be read.

Do you have them in SD card? Maybe the SD card could not be read when starting the app?
Internal downloaded maps or external opened map files?

Map layers are defined by the map theme.
Changing the map theme, the map layers are set again from the new map theme.

**Reply by DolomitesTom** [reply]
An online map was displayed. Internal download, maps on SD card, no problems with the card at all since years.

Ok thanks

**Reply by devemux86** [reply]
> An online map was displayed

@DolomitesTom thanks, then it is indeed the case that the offline map files could not be read for some reason,
so the online map was activated.
